### PR TITLE
Edit pass on ASP.NET Core metrics

### PR DESCRIPTION
**Pull request by Rick-Anderson**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/diagnostics/built-in-metrics-aspnetcore.md](https://github.com/dotnet/docs/blob/6b7629200453ae7f7cd297f0c8bccdf75de4adcf/docs/core/diagnostics/built-in-metrics-aspnetcore.md) | [ASP.NET Core Metrics](https://review.learn.microsoft.com/en-us/dotnet/core/diagnostics/built-in-metrics-aspnetcore?branch=pr-en-us-37529) |

### CODE REVIEW

File: docs/core/diagnostics/built-in-metrics-aspnetcore.md [hunk 1]
```diff
@@ -146,7 +154,7 @@ Available staring in: ASP.NET Core 8.0
 
 | Name     | Instrument Type | Unit (UCUM) | Description    |
 | -------- | --------------- | ----------- | -------------- |
-| `aspnetcore.rate_limiting.queued_requests` | UpDownCounter | `{request}` | Number of requests that are currently queued, waiting to acquire a rate limiting lease. |
+| `aspnetcore.rate_limiting.queued_requests` | UpDownCounter | `{request}` | Number of requests that are currently queued waiting to acquire a rate limiting lease. |
```
Comment (Rick-Anderson) [inline docs/core/diagnostics/built-in-metrics-aspnetcore.md:157]: @tdykstra  I think this reads better without the `,`

Comment (tdykstra) [reply docs/core/diagnostics/built-in-metrics-aspnetcore.md:157]: I think it needs the comma.

Comment (IEvangelist) [reply docs/core/diagnostics/built-in-metrics-aspnetcore.md:157]: I like the comma too.

**Review by IEvangelist — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@IEvangelist should we move this doc to the https://github.com/dotnet/AspNetCore.Docs repo and fix up your TOC or just leave it here and the ASP.NET Core team can own it? It might make more sense to leave it here.

cc @noahfalk  if he has a strong opinion.

**Pull request merged**